exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporterBuilder.java

```diff
@@ -105,7 +105,7 @@ public OtlpGrpcSpanExporterBuilder setCompression(String compressionMethod) {
 
   /**
    * Sets the certificate chain to use for verifying servers when TLS is enabled. The {@code byte[]}
-   * should contain an X.509 certificate collection in PEM format. If not set, TLS connections will
+   * should contain a X.509 certificate collection in PEM format. If not set, TLS connections will
```

Comment:
This was correct before. English is weird, but since you pronounce it "ex", you need an "an" here.